README.md

```diff
@@ -61,8 +49,8 @@ From source repository:
 > pip install .
```

Comment:
What do you think about removing the leading `>` characters from the bash/shell lines?
- make consistent with the code sections on lines 67 and 86
- makes for a cleaner copy/paste to a user terminal (that `>` character comes along for the ride)

Also, standardize on "bash" or "shell" for syntax hightlighting, perhaps. Or maybe just "console"?